help/deploy/aws/cloudformation/templates/fargate.template.yml

```diff
@@ -1481,10 +1481,10 @@ Resources:
       - CvmTCP80Listener
       - CvmTCP443Listener
       - CvmTCP8090IntListener
-      - DatagrokJupyterKernelGatewayTCP5005ListenerRule
-      - DatagrokJupyterKernelGatewayTCP5005IntListenerRule
-      - DatagrokJupyterKernelGatewayTCP8888ListenerRule
-      - DatagrokJupyterKernelGatewayTCP8888IntListenerRule
+      - DatagrokJupyterTCP5005ListenerRule
+      - DatagrokJupyterTCP5005IntListenerRule
+      - DatagrokJupyterTCP8888ListenerRule
+      - DatagrokJupyterTCP8888IntListenerRule
```

Comment:
I would use previous names instead. Cause we have 2 services and one common name can be confusing

Reply:
o no, I've done so much to do The One name from two...